message_adapter/message_adapter.py

```diff
@@ -379,21 +375,21 @@ def __assignJsonPathValue(self, message, jspath, value):
         * @param {*} message The message to be update
         * @return {*} updated message
         """
-        if len(parse(jspath).find(message)) > 0:
-            parse(jspath).update(message, value)
-        else:
+        if not parse(jspath).find(message):
```

Comment:
Only the changes below this line in this file are relevant to the ticket. The rest were minor pylint changes that I started making because VScode was flagging them. But I can revert those if desired